processor/src/main/resources/org/mapstruct/ap/internal/model/MethodReference.ftl

```diff
@@ -20,7 +20,16 @@
 -->
 <@compress single_line=true>
     <#-- method is either internal to the mapper class, or external (via uses) declaringMapper!=null -->
-    <#if declaringMapper??><#if static><@includeModel object=declaringMapper.type/><#else>${mapperVariableName}</#if>.</#if>${name}<#if (parameters?size > 0)>( <@arguments/> )<#else>()</#if>
+    <#if declaringMapper??><#if static><@includeModel object=declaringMapper.type/><#else>${mapperVariableName}</#if>.<@params/>
+    <#elseif static && mapperToImplement??><@includeModel object=mapperToImplement/>.<@params/>
```

Comment:
Yes, that'd be nice. We'd just have to make sure that the generated processor JAR still works with Java 7 ((some of) the integration tests should make that sure as they also run with JDK 7 when configured properly).